mandatory/2-function-creation.js

```diff
@@ -15,15 +25,26 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+var evenAndEqual = true
+var notEvenAndEqual = false
+
+function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num <= 100) {
+    return evenAndEqual}
+    else {
+      return notEvenAndEqual
+    }
+  }
+
 
 /* 
 Write a function that returns a copy of the given array arr, but with the element at the given index, index removed.
 The function must NOT change the original array, arr.
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  arr.splice(index, 1)
+  return arr; // complete this statement
```

Comment:
The function must NOT change the original array, arr. Does "splice" change the original array? And how can we return a new array with result, and leave input arr untouched?